examples/passing/DctorName.purs

```diff
@@ -0,0 +1,11 @@
+module Main where
+
+import Control.Monad.Eff.Console (log)
+
+newtype Bar' = Bar' Int
+
+data Foo' = Foo' Bar'
+
+data Baz'' = Baz''
+
+main = log "Done"
```

Comment:
Thanks, this has caught the case where the parser for the constructor binding in case statements did not deal with the symbols. I have updated the parser and example.